### ISSUE TITLE
convolution layers of keras and caffe has different output

### ISSUE THREAD

**Issue by xiaofeiwu** [uhfband/keras2caffe#3]
It seems that the output of convolution layers of keras and caffe has different output even when the input the and weights are all the same. It may be caused by the padding implementation in tensorflow and caffe. Does anyone meet the same issue?

**Comment by nerddd** [comment]
Yes，I met the same problem.Did u solve the issue?

**Comment by nerddd** [comment]
I think pooling layer has the difference between caffe and tf

**Comment by to-be-snail** [comment]
> It seems that the output of convolution layers of keras and caffe has different output even when the input the and weights are all the same. It may be caused by the padding implementation in tensorflow and caffe. Does anyone meet the same issue?

I met the same question.Did you solve the problem?

**Comment by to-be-snail** [comment]
> I think pooling layer has the difference between caffe and tf

I met the same question.Did you solve the problem?

**Comment by DanielXu123** [comment]
So, bascially, the transfered caffemodel can not get the same results as the keras .h5 model predict?
